app/components/product/alias-form.js

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { isPresent } from '@ember/utils';
+
+// https://youmightnotneed.com/lodash/#without
+// eslint-disable-next-line
+const without = (arr, ...args) => arr.filter(item => !args.includes(item))
+
+export default class AliasFormComponent extends Component {
+  @tracked newAlias;
+
+  get newAliasIsValid() {
+    return isPresent(this.newAlias);
+  }
+
+  @action
+  addAlias(newAlias) {
+    if (newAlias) {
+      const aliases = this.args.aliases.slice();
+      aliases.push(newAlias);
+      this.args.onChange(aliases);
+      this.newAlias = '';
```

Comment:
Out of curiosity (I see others doing it as well sometimes), why reset it on an empty string and not on `null` or `undefined`?

Reply:
I think, given that an empty string is falsy too, the result when testing for emptyness is the same. Compared to `null`, the empty string has the (admittedly very small) added value of doing some "documentation" that says "this is supposed to be a string". Poor man's typescript you might say. Not a reason to be inconsisten though. I'm changing it.